WebRoot/css/zm.css

```diff
@@ -6585,3 +6579,19 @@ a.ZmLinkDisabled {
 	@MailAttachmentsAllText@
 }
 
+/* Create Contact Group dialog */
+#CreateContactGroupDialog {
+	@CreateContactGroupDialog@
+}
+
+#CreateContactGroupDialog table.ZPropertySheet {
+	@CreateContactGroupDialogTable@
+}
+
+#CreateContactGroupDialog table.ZPropertySheet td {
+	@CreateContactGroupDialogCell@
+}
+
+#CreateContactGroupDialog table.ZPropertySheet td input[type='text'] {
```

Comment:
There are very less chances getting another input field. Thought was to have a simple selector . But we can discuss more.